0000-template.md

```diff
@@ -8,14 +8,26 @@ One paragraph explanation of the feature.
 
 # Motivation
 
-Why are we doing this? What use cases does it support? What is the expected outcome?
+Why are we doing this? What use cases does it support? What is the expected
+outcome?
 
 # Detailed design
 
-This is the bulk of the RFC. Explain the design in enough detail for somebody familiar
-with the framework to understand, and for somebody familiar with the implementation to implement.
-This should get into specifics and corner-cases, and include examples of how the feature is used.
-Any new terminology should be defined here.
+This is the bulk of the RFC. Explain the design in enough detail for somebody
+familiar with the framework to understand, and for somebody familiar with the
+implementation to implement. This should get into specifics and corner-cases,
+and include examples of how the feature is used. Any new terminology should be
+defined here.
+
+# Programming model
+
+How does this proposal impact the existing Ember programming model? What
+names and terminology work best for these concepts and why?
+
+# Pedagogy
+
+How should this feature and introduced and taught to new and existing Ember
+users?
 
 # Drawbacks
 
```

Comment:
So many RFCs put "N/A" here, which is bullshit. We should include a note that this should not be blank, and the more thought put into it, the better.